cu-webteam/d8-platform
# Link featured content

# Description

Using featured content on front page. Fixed the bug where a linked header altered the css. Added css to handle styling responsively on front page


## Steps to Test or Reproduce
git fetch link-featured-content
vagrant up
vagrant ssh
visit local.creighton.edu
if one is not created make a featured group with three items then add it to the front page
- to test the bug fix at least one item should have a linked header and another should not,  though look the same
analyze how layout changes as browser changes.
